servicetalk-examples/http/serialization/json/src/main/java/io/servicetalk/examples/http/serialization/json/blocking/streaming/BlockingPojoStreamingClient.java

```diff
@@ -36,11 +35,9 @@ public static void main(String[] args) throws Exception {
                             new CreatePojoRequest("value2"),
                             new CreatePojoRequest("value3")),
                             REQ_STREAMING_SERIALIZER));
-            System.out.println(response);
-            try (BlockingIterator<PojoResponse> payload = response.payloadBody(RESP_STREAMING_SERIALIZER).iterator()) {
```

Comment:
I am curious about the removal of several try-with-resources for payload bodies. Is the change neutral for resource management?

Reply:
good point, tried to simplify the example, reverted with a comment: https://github.com/apple/servicetalk/pull/2106/commits/0e630906cb516a8709033d3b580fd20af1b27879